src/cftkb/discipad/discipad.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Discipad",
+  "vendorId": "0x6B62",
+  "productId": "0x1769",
+  "lighting": "none",
+  "matrix": {"rows": 5, "cols": 4},
+  "layouts": {
+    "keymap": [
+      ["0,1","0,2","0,3","0,4"],
+      ["1,0","1,1","1,2",{h:2},"2,3"],
+      ["2,0","2,1","2,2"],
+      ["3,0","3,1","3,2",{c:"#777777",h:2},"4,3"],
+      [{c:"#cccccc",w:2},"4,0","4,2"]
+    ]
```

Comment:
This looks like copy-paste out of KLE raw pane, which is invalid JSON for VIA... need to save as JSON file first.